e2e_tests/tests/cluster/test_workspace_org.py

```diff
@@ -426,3 +427,60 @@ def test_reset_workspace_checkpoint_storage_conf() -> None:
         assert resp_patch.workspace.checkpointStorageConfig is None
     finally:
         _delete_workspace_and_check(sess, resp_w.workspace)
+
+
+@contextlib.contextmanager
+def setup_workspace(session: api.Session) -> Generator[bindings.v1Workspace, None, None]:
```

Comment:
cc @nrajanee this is a useful helper for tests

Reply:
I was thinking of having it set up `n` workspaces to avoid having to nest multiple of these if you need more than one. should be easy to go from this to that version.

Reply:
thanks! Once this is merged I can use this in my tests.